neptune/internal/backends/alpha_integration_backend.py

```diff
@@ -174,39 +194,64 @@ def get_system_channels(self, experiment):
             # pylint: disable=protected-access
             raise ExperimentNotFound(
                 experiment_short_id=experiment.id, project_qualified_name=experiment._project.full_id)
+
         return [
-            AlphaChannelWithLastValue(
-                ch_id=attr.stringSeriesProperties.attributeName,
-                # ch_name is ch_id without first namespace
-                ch_name=attr.stringSeriesProperties.attributeName.split('/', 1)[-1],
-                ch_type=attr.stringSeriesProperties.attributeType,
-            )
-            for attr in experiment.attributes
-            if (attr.type == AlphaAttributeType.STRING_SERIES.value
-                and attr.name.startswith(alpha_consts.MONITORING_ATTRIBUTE_SPACE))
+            AlphaChannelDTO(attr) for attr in experiment.attributes
+            if AlphaChannelDTO.is_valid_attribute_for_channel(attr)
         ]
 
+    @with_api_exceptions_handler
+    def get_channels(self, experiment) -> Dict[str, AlphaChannelDTO]:
+        api_channels = [
+            channel for channel in self._get_channels(experiment)
+            # return channels from LOG_ATTRIBUTE_SPACE namespace only
+            if channel.id.startswith(alpha_consts.LOG_ATTRIBUTE_SPACE)
+        ]
+        channels = dict()
+        for ch in api_channels:
```

Comment:
This code is analogous to [this experiment methods](https://github.com/neptune-ai/neptune-client/pull/376/files#diff-f9469dadddb4d894b4ab0167d6248ad4208edceaeb7f04f8e2a4c6791625b150L274).
But we're relying on `ch.lastX` alone. Do you think that we may need some alpha  equivalent of `channels_last_values_by_name`?

Reply:
> So, if I am correct, old code returned only USER channels, but y value was filled only for NUMERIC or TEXT channels.
> 
> Your code seems to operate on AlphaChannelDTO without value. Isn't it a problem?

No `AlphaChannelDTO` has `lastX` value which returns `last` of stored attribute (which is one of: `StringSeriesAttributeDTO`, `FloatSeriesAttributeDTO` or `ImageSeriesAttributeDTO`).
I've just noticed that `ImageSeriesAttributeDTO` doesn't have mentioned `last` value, so it's matching with old client behaviour.

I think that current implementation of `AlphaChannelDTO.lastX` may be good equivalent of `channels_last_values_by_name` in old client.

Reply:
`last` in api attribute is ment to be value, not a step (x). Unfortunatelly lastStep is missing in api.
I don't think it is important to have last step here. Is it used by our code or does it exists only to be returned to user? If we don't need it let's just add a bug to Jira and set it to None. However, it is important to return value to the user. So `AlphaChannelDTO` should have last value as well.

Reply:
Right...
I've renamed `lastX` to `lastY` in `AlphaChannelDTO` - it's additional field we may use for initializing `ch.y`. And I've updated `lastX` to always return None.
Right now code works, but I've created issue for fixing problem with missing X.